src/accessories/CampaignImporterExporter.ttslua

```diff
@@ -0,0 +1,302 @@
+    spawnObjectData({data = importData[2]}) 
+    --create chaos bag
+    spawnObjectData({data = importData[3]})
+    --populate trauma values
+    if importData[4] then
+        updateCounters(importData[4])
+    end
+    --populate ArkhamDB deck IDs
+    if importData[5] then
+        deckImporter.call("updateDeckIds", importData[5])
+    end
+    --set investigator count
+    investigatorCounter.call("updateVal", importData[6])
+    --set campaign guide page
+    if findCampaignGuide() then
+        Wait.condition(
+            function()
+                --log("Campaign Guide import successful!")
+            end,
+            function()
+                findCampaignGuide().Book.setPage(importData[7])
+                return findCampaignGuide().Book.getPage() == importData[7]
+            end,
+            1,
+            function()
+                --log("Campaign Guide import failed!")
+            end
+        )
+    end
+    Global.call("loadSettings", importData[8])
+    broadcastToAll("Campaign successfully imported!", Color.Green)
+end
+
+function createCampaignToken(_, _, _)
+    local campaignBoxGUID = ""
+    -- find active campaign
+    for _, obj in ipairs(getObjectsWithTag("CampaignBox")) do
+        if obj.type == "Bag" and #obj.getObjects() == 0 then
+            campaignBoxGUID = obj.getGUID()
+        end
+    end
+    if campaignBoxGUID == "" then
+        broadcastToAll("Campaign box with all placed objects not found!", Color.Red)
+        return
+    end
+    local campaignLog = findCampaignLog()
+    if campaignLog == nil then
+        broadcastToAll("Campaign log not found!", Color.Red)
+        return
+    end
+    local chaosBag = findChaosBag()
+    if chaosBag == nil then
+        broadcastToAll("Chaos bag not found in standard position!", Color.Red)
+        return
+    end
+    local counterData = campaignLog.getVar("ref_buttonData")
+    if counterData ~= nil then
+        traumaValues = {}
+        printToAll("Trauma values found in campaign log!", "Green")
+        for i = 1, 10, 3 do
+            traumaValues[1 + (i - 1) / 3] = counterData.counter[i].value
+            traumaValues[5 + (i - 1) / 3] = counterData.counter[i + 1].value
+        end
+    else
+        printToAll("Trauma values could not be found in campaign log!", "Yellow")
+        printToAll("Default values for health and sanity loaded.", "Yellow")
+    end
+    deckIds = {deckImporter.getInputs()[1].value, deckImporter.getInputs()[2].value, deckImporter.getInputs()[3].value, deckImporter.getInputs()[4].value}
+    local campaignGuide = findCampaignGuide()
+    if campaignGuide == nil then
+        broadcastToAll("Campaign loguideg not found!", Color.Red)
+        return
+    end
+    local campaignGuidePage = campaignGuide.Book.getPage()
+
+    campaignData = {
+        campaignBoxGUID,
+        campaignLog.getData(),
+        chaosBag.getData(),
+        traumaValues,
+        deckIds,
+        investigatorCounter.getVar("val"),
```

Comment:
This is already exposed via the PlayAreaApi.
https://github.com/argonui/SCED/blob/fc68103f6748b4b85b045a8b0d70e1dd3c4d26cc/src/core/PlayAreaApi.ttslua#L8